code/script.js

```diff
@@ -34,18 +39,139 @@ const showMessage = (message, sender) => {
 }
 
 // Starts here
+//Question number 1
 const greeting = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage("Greetings traveler, what is your name?", 'bot');
+}
+
+//User answer 1 
+//This shows a bubble with the users name after they've written it
+const handleNameInput = (event) => {
+  event.preventDefault();
+  const name = nameInput.value;
+  showMessage(name, 'user');
+  nameInput.value = '';
+  setTimeout(() => chooseTourGuide(name), 750);
+}
+
+// Question numer 2 
+const chooseTourGuide = (name) => {
+  showMessage(`${name}, please choose your tour guide.`, 'bot');
+  inputWrapper.innerHTML=` 
+  <button id="pirateBtn" type="submit">Pirate</button>
+  <button id="vampireBtn" type="submit">Vampire</button>
+  <button id="witchBtn" type="submit">Witch</button>`
+  document
+  .getElementById('pirateBtn')
+  .addEventListener('click',()=> handleTourGuide ('Pirate'));
+
+  document
+  .getElementById('vampireBtn')
+  .addEventListener('click',()=> handleTourGuide ('Vampire'));
+
+  document
+  .getElementById('witchBtn')
+  .addEventListener('click',()=> handleTourGuide ('Witch'));
+}
+
+//document.getElementById('bot-icon').src ="assets/pirate-icon.svg";
+// user answer 2, bubble with chosen tour guide
+const handleTourGuide = (tourGuide) => {
+  showMessage(tourGuide, 'user');
+  setTimeout(() => chooseActivity(tourGuide), 750);
+}
+
+// Question numer 3, drop down menu for choices
+const chooseActivity = (tourGuide) => {  
+  showMessage (`You have chosen the ${tourGuide}. What do you want to do together?`, 'bot');
+  if (tourGuide === 'Pirate') {
+    document.body.style.backgroundImage = "url('assets/pirate-ship.jpg')"
+    // "Pirate Ship. Nikon D3100. DSC_0170." by Robert.Pittman is licensed under CC BY-ND 2.0. 
```

Comment:
This part is really cool and creative. Also, from this we can learn that it's possible to do this way.